### DISCUSSION TITLE
Multiple solution is too little

### DISCUSSION THREAD

**Discussion by MikeKen-Ken** [community/community#37304] [Copilot Conversations] [answered]
Hello,
  I have subscribed Copilot, i found multiple solution is too little, only ten at a time, can it be configured to support more at a time?
Thank you.

**Comment by iamcjwilliams** [comment, marked as answer]
@MikeKen-Ken Which IDE are you using? Here is a snippet for vscode:

>  "github.copilot.advanced": {
>         "listCount": <_however many you want_>
>     },

**Reply by MikeKen-Ken** [reply]
Thank you very much for your answer, I probably understand how to configure it. What are the advanced settings that can be configured, and is there any documentation? Thank you very much!

**Reply by iamcjwilliams** [reply]
Here is a link to the [documentation](https://docs.github.com/en/copilot)

**Reply by MikeKen-Ken** [reply]
I can find this link, but I can't find the documentation on advanced setting. Is there any documentation about advanced setting? Thank you!
